source/getting-started/glossary.md

```diff
@@ -0,0 +1,85 @@
+Joining a web development community can be a challenge within itself, especially when all the resources you visit assume you're familiar with other technologies that you're not familiar with.
+
+Our goal is to help you avoid that mess and come up to speed as fast as possible; you can consider us your internet friend.
+
+## CDN
+Content Delivery Network
+
+This is typically a paid service you can use to get great performance for your app. Many CDNs act as caching proxies to your origin server; some require you to upload your assets to them. They give you a URL for each resource in your app. This URL will resolve differently for folks depending on where they're browsing.
```

Comment:
"caching proxy" is not a great term if we're trying to educate people who don't have a lot of context. "They give you a URL for each resource [which] will resolve differently" - I don't think this really explains what CDNs do very well.